package.json

```diff
@@ -28,17 +33,26 @@
     "shortstop": "^1.0.3"
   },
   "devDependencies": {
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "commitizen": "^4.2.4",
+    "cz-conventional-changelog": "^3.3.0",
     "eslint": "^5.15.1",
     "eslint-config-hapi": "^10.0.0",
     "eslint-plugin-hapi": "^4.0.0",
     "nyc": "^14.1.1",
+    "semantic-release": "^18.0.1",
     "tape": "^4.2.2"
   },
   "scripts": {
+    "commit": "cz",
     "cover": "nyc npm test",
     "lint": "eslint lib",
     "test": "npm run lint && ENV_TEST=5678 tape test/*.js"
   },
-  "main": "./lib",
-  "license": "MIT"
+  "config": {
```

Comment:
Looks good to me.    for consistency,   please  put. dependencies sections  to  the bottom, and  put config  section  below scripts section.  Here is the steerage repo for  reference. 

https://github.com/ExpediaGroup/steerage/blob/master/package.json